assets/src/components/settings/global/GlobalSettingsAIProviders.tsx

```diff
@@ -94,19 +95,74 @@ export function validateAttributes(
   }
 }
 
-export function OpenAIAnthropicSettings({
+export function OpenAISettings({
   enabled,
   settings,
   updateSettings,
 }: {
   enabled: boolean
-  settings: AiSettingsAttributes['openai'] | AiSettingsAttributes['anthropic']
+  settings: AiSettingsAttributes['openai']
   updateSettings: (
-    update: NonNullable<
-      Partial<
-        AiSettingsAttributes['openai'] | AiSettingsAttributes['anthropic']
+    update: NonNullable<Partial<AiSettingsAttributes['openai']>>
+  ) => void
+}) {
+  const theme = useTheme()
+
+  return (
+    <>
+      <FormField
+        label="Model"
+        hint="Leave blank for Plural default."
+        flex={1}
+      >
+        <Input
+          disabled={!enabled}
+          value={settings?.model}
+          onChange={(e) => {
+            updateSettings({ model: e.currentTarget.value })
+          }}
+        />
+      </FormField>
+      <FormField
```

Comment:
This field should be optional, will the form still enable saving if blank?

Reply:
@michaeljguarino Yes it does.

Reply:
Only access token is required for OpenAI at the moment.